src/api/java/com/minecolonies/api/items/ItemBlockHut.java

```diff
@@ -47,7 +47,7 @@ public void addInformation(@NotNull ItemStack stack, World worldIn, @NotNull Lis
         super.addInformation(stack, worldIn, tooltip, flagIn);
         if (block.needsResearch())
         {
-            tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_1, block.getName()));
+            tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_1, block.getTranslatedName()));
             tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_2, block.getName()));
```

Comment:
Should both of these be translated?

Reply:
Only HUT_NEEDS_RESEARCH_TOOLTIP_1 ("§cThe %s requires research") actually supports arguments, the second is just "§cat the University to be built."  I could see a point where people would want a way to get the block's id as used in the argument, and using a lang file to do so seems reasonable.  But that could also be done other ways, or using translatedName as the first argument as Name as the second.

Reply:
Yeah, I'd update this to pass the parameters you expect somebody might need to both of these, even if we're not using them today.